<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>766</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Rubik</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Attendance</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../logo.png</normaloff>../logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="attendanceLabel">
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Attendance</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarFrom">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QCalendarWidget QMenu:{
background-color: rgb(0, 0, 0);
}
</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="minimumDate">
           <date>
            <year>2021</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="gridVisible">
           <bool>true</bool>
          </property>
          <property name="horizontalHeaderFormat">
           <enum>QCalendarWidget::ShortDayNames</enum>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
          <property name="navigationBarVisible">
           <bool>true</bool>
          </property>
          <property name="dateEditEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateFrom">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarTo">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="minimumDate">
           <date>
            <year>2021</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="gridVisible">
           <bool>true</bool>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateTo">
          <property name="font">
           <font>
            <family>OCR A Extended</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="attendanceTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{
border-radius: 10px}</string>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>220</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchAttendance">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(156, 120, 255);
border-radius: 15px;
	color: rgb(255, 255, 255);
}

QPushButton:hover {
background-color:rgb(170, 145, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}
QPushButton:pressed {
background-color:rgb(125, 96, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}

</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="backToMenu">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color:rgb(156, 120, 255);
border-radius: 15px;
color: rgb(255, 255, 255);

}

QPushButton:hover {
background-color:rgb(170, 145, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}
QPushButton:pressed {
background-color:rgb(125, 96, 204);
border-radius: 15px;
color: rgb(255, 255, 255);

}


</string>
      </property>
      <property name="text">
       <string>Back to Menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../logo.qrc"/>
 </resources>
 <connections/>
</ui>
